corehq/apps/linked_domain/tests/test_update_roles.py

```diff
@@ -68,49 +56,48 @@ def test_update_report_list(self):
         self.assertEqual(roles['other_test'].assignable_by, [roles['test'].get_id])
 
     def test_match_names(self):
-        self.assertEqual([], UserRole.by_domain(self.linked_domain))
+        self.assertEqual([], SQLUserRole.objects.get_by_domain(self.linked_domain))
 
-        role = UserRole(
-            domain=self.linked_domain,
-            name='other_test',
-            permissions=Permissions(
-                edit_web_users=False,
-                view_locations=True,
-            ),
-            assignable_by=[self.role.get_id],
+        # create role in linked domain with the same name but no 'upstream_id'
+        SQLUserRole.create(
+            self.linked_domain, 'other_test', Permissions(edit_web_users=True, view_locations=True)
         )
-        role.save()
 
         update_user_roles(self.domain_link)
-        roles = {r.name: r for r in UserRole.by_domain(self.linked_domain)}
+        roles = {r.name: r for r in SQLUserRole.objects.get_by_domain(self.linked_domain)}
         self.assertEqual(2, len(roles))
         self.assertTrue(roles['other_test'].permissions.edit_web_users)
         self.assertEqual(roles['other_test'].upstream_id, self.other_role.get_id)
 
     def test_match_ids(self):
-        self.assertEqual([], UserRole.by_domain(self.linked_domain))
+        self.assertEqual([], SQLUserRole.objects.get_by_domain(self.linked_domain))
 
-        role = UserRole(
-            domain=self.linked_domain,
-            name='id_test',
-            permissions=Permissions(
-                edit_web_users=False,
-                view_locations=True,
-            ),
-            assignable_by=[self.role.get_id],
+        # create role in linked domain with upstream_id and name not matching upstream name
+        SQLUserRole.create(
+            self.linked_domain, 'id_test', Permissions(edit_web_users=True, view_locations=True),
```

Comment:
Is it intended that `edit_web_users` changed values? DOesn't look like it matters to the test, just curious.

Reply:
well spotted. It should change since that's how we know it was actually updated: 28a614530af